lib/vtls/x509asn1.c

```diff
@@ -552,6 +559,15 @@ static CURLcode GTime2str(struct dynbuf *store,
                        sep, (int)tzl, tzp);
 }
 
+#ifdef UNITTESTS
+/* used by unit1489.c */
```

Comment:
```suggestion
/* used by unit1656.c */
```

Reply:
Right, needed to use another number and now there are relics. Thanks for catching this.

Reply:
maybe commit this correction at least before merge?